cmake/TaichiCore.cmake

```diff
@@ -51,28 +50,25 @@ include_directories(external/spdlog/include)
 set(LIBRARY_NAME ${CORE_LIBRARY_NAME})
 
 if (TI_WITH_OPENGL)
-  if(NOT GLEW_VERSION)
-    set(GLEW_VERSION 2.0.0)
-  endif()
-  find_package(GLEW ${GLEW_VERSION})
-  if (GLEW_FOUND)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DTI_WITH_OPENGL")
-    message("Building with GLEW ${GLEW_VERSION}")
-    message("Using GLEW: ${GLEW_LIBRARIES}")
-    target_include_directories(${LIBRARY_NAME} PUBLIC ${GLEW_INCLUDE_DIRS})
-    target_link_libraries(${LIBRARY_NAME} ${GLEW_LIBRARIES} GLEW)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGLEW_STATIC")
-
-    set(GLFW_BUILD_DOCS OFF CACHE BOOL "" FORCE)
-    set(GLFW_BUILD_TESTS OFF CACHE BOOL "" FORCE)
-    set(GLFW_BUILD_EXAMPLES OFF CACHE BOOL "" FORCE)
-
-    message("Building with GLFW")
-    add_subdirectory(external/glfw)
-    target_link_libraries(${LIBRARY_NAME} glfw)
```

Comment:
What I find is that adding this line enforces `cmake` to build GLFW for `libglfw3.a`.

Reply:
You find this enfocing on your machine? Did you install glfw in your /usr/include?

Reply:
Yes, and I removed `usr/lib/libglfw3.a`. Maybe you can try it on your Arch machine as well.